ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/CompressionMetrics.scala

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.appendonlydao.events
+
+import com.codahale.metrics.Histogram
+import com.daml.metrics.Metrics
+
+object CompressionMetrics {
```

Comment:
Is that a file that was modified in parallel to this PR? It looks very different to its original in `dao`

Reply:
Not only this, but also the CompressionStrategy is quite different. As part of the PoC I had to rebase at some point, and I faced all changes regarding compression. There I wanted to have compression easily used in the new PoC indexer, and therefore I made refactorings to the original compression supporting code. I think the refactoring makes sense, and I would go ahead with that if that's fine with you.
Something else to note: the store.serialisation.Compression code was not changing much during my Compression refactorings, so I kept that around (was not forking that as well).
On that account: I forgot to add there a TODO to cleanup that helper method which is not needed any more...just did that as well.

Reply:
Thanks for the explanation. It makes sense